build-tools/packages/build-cli/src/config.ts

```diff
@@ -63,6 +63,22 @@ export interface FlubConfig {
 	 * Configuration for the `generate:releaseNotes` command.
 	 */
 	releaseNotes?: ReleaseNotesConfig;
+
+	/**
+	 * The multiple of minor versions to use for calculating the next version in the legacy compatibility range.
+	 */
+	releaseReport?: LegacyCompatInterval;
+}
+
+/**
+ * Represents legacy compat interval per release groups.
+ */
+export interface LegacyCompatInterval {
+	/**
+	 * Each key in the `legacyCompatInterval` object represents a specific release group,
+	 * and the associated value is a number that defines the legacy compatibility interval for that group.
```

Comment:
Per use in `generateReleaseReport` I think the keys should be groups OR package names. A package not part of group could have legacy compat versioning.